### ISSUE TITLE
Compilation breaks if 'summary' has several paragraphs

### ISSUE THREAD

**Issue by r03ert0** [brainhack-proceedings/template#1]
My summary section had 3 paragraphs, and compilation didn't work. Making a single paragraph make compilation work again. If a single paragraph is expected, that should be said in the doc, or in the comments inside the template file. Probably, a maximum number of characters should be indicated, otherwise the layout of the webpage may break.

**Comment by agahkarakuzu** [comment]
@r03ert0 yes, summary section requires a single paragraph as it is a yaml field. One trick would be passing `<br>` between paragraphs, on the other hand, I like single paragraphs as a summary :)

**Comment by r03ert0** [comment]
Sure. We could make a single paragraph summary mandatory, but that would have to be made explicit. Otherwise, people will have broken compilations and it'll be tricky to track the bug...
Now, the single paragraph summary may be a matter of preference, and some other people may like to have several paragraphs. In particular, medical people like to separate their summary paragraphs into "background/methods/results" and stuff...
